packages/zambdas/src/shared/pdf/external-labs-order-form-pdf.ts

```diff
@@ -421,7 +423,9 @@ export async function createExternalLabsOrderFormPDF(
 
   console.debug(`Created external labs order form pdf bytes`);
   const bucketName = 'visit-notes';
-  const fileName = 'ExternalLabsOrderForm.pdf';
+  const fileName = `ExternalLabsOrderForm-${getLabFileName(input.orderName || '')}-${DateTime.fromISO(
```

Comment:
If there is no order name, this is going to evaluate to look like it has a long dash `ExternalLabsOrderForm--<date>`. Would suggest changing it to something like
```suggestion
  const fileName = `ExternalLabsOrderForm-${input.orderName ? getLabFileName(input.orderName) + '-' : ''}${DateTime.fromISO(
```

Reply:
When will orderName not have a value?